src/components/GroupedData.tsx

```diff
@@ -20,122 +20,141 @@ function GroupedData() {
-                                stroke="#000000"
-                                strokeWidth="1"
-                                stroke-linecap="round"
-                                stroke-linejoin="round"
-                              />
-                            </svg>{" "}
-                          </div>
-                        ) : (
-                          ""
-                        )}
-                      
-                        <p>CO_DESIGN_LEVEL: {level} </p>
-                        {hoveredIndex === dataIndex && (
-                          <div className="absolute bg-white border p-4 mt-2 z-10">
-                            <p>Extra: {itemdata.Extra}</p>
-                            <p>{itemdata.Manual_Tags}</p>
-                          </div>
-                        )}
-                      </div>
-                    );
-                  }
-                })}
-              </div>
-            </div>
-          );
-        })}
+      <div
+        ref={gridContainerRef}
+        className="grid grid-cols-1 gap-5  p-20 w-full"
+      >
+        <ArcherContainer
+          offset={20}
+          strokeDasharray="."
+          noCurves
+          strokeColor-="black"
+        >
+          <div className="grid grid-cols-2 gap-10">
+            {uniqueThemes.map((themItem, index) => {
+              return (
+                <div
+                  key={themItem}
+                  id={themItem}
+                  className="bg-gray-100 border border-dashed"
+                >
+                  <div className="bg-yellow-200 flex justify-between px-5 py-5 w-4/5">
+                    <h3 className="text-2xl font-semibold w-1/3-">
+                      {themItem}
+                    </h3>
+                    <p className="text-sm w-1/2- text-center">
+                      Engaging with entire systems approaches requires
+                      humanitarian practitioners to challenge and rethink how to
+                      interact with humanitarian systems that surround them.
+                      This outlines what needs to change and how.
+                    </p>
+                  </div>
+                  <div className=" m-5 grid grid-cols-2 gap-x-64 gap-y-20 py-10">
+                    {data.map((data: any) => {

```

Comment:
data type instead of being any we can use :
```
export interface ParsedManualTags {
  CORE_TOOLKIT?: boolean;
  CO_DESIGN_LEVEL?: string;
  ["USEFUL FOR"]?: string[];
  ["PROJECT STEP"]?: string;
  THEME?: string;
  ACCESS?: string;
  TOOLS?: string;
}

export interface Entry {
  Key: string;
  Title: string;
  Extra: string | null;
  Manual_Tags: string;
  PARSED_MANUAL_TAGS: ParsedManualTags;
  PARSED_RELATES_TO?: string[];
}

export type Entries = Entry[];
```